backend/src/v2/component/launcher_v2.go

```diff
@@ -507,6 +516,15 @@ func execute(
 		writer = os.Stdout
 	}
 
+	// If TLS enabled, append mounted custom CA to system CA and save to temp file for executor access.
+	if tlsEnabled {
```

Comment:
This is only necessary when CaCertPath is set. It's valid to have TLS enabled without needing a custom CA to trust.